README.md

```diff
@@ -1,15 +1,245 @@
-# Pipeline name
+# Somatic-shortV-nf
+
+<p align="center">
+:wrench: This pipeline is currently under development :wrench:
+</p> 
+
+  - [Description](#description)
+  - [Diagram](#diagram)
+  - [User guide](#user-guide)
+      - [Infrastructure usage and
+        recommendations](#infrastructure-usage-and-recommendations)
+  - [Benchmarking](#benchmarking)
+  - [Workflow summaries](#workflow-summaries)
+      - [Metadata](#metadata)
+      - [Component tools](#component-tools)
+  - [Additional notes](#additional-notes)
+  - [Help/FAQ/Troubleshooting](#helpfaqtroubleshooting)
+  - [Acknowledgements/citations/credits](#acknowledgementscitationscredits)
+
+## Description
+Somatic-shortV-nf is a Nextflow pipeline for identifying somatic short variant events in Illumina short read whole genome sequence data. 
+We have followed the [GATK Best practices](https://gatk.broadinstitute.org/hc/en-us/articles/360035894731-Somatic-short-variant-discovery-SNVs-Indels-) .  
```

Comment:
Use the description already provided in the [SomaticShortV](https://github.com/Sydney-Informatics-Hub/Somatic-ShortV#somatic-shortv) README. Just remove the text about being optimised for Gadi and replace with a comment about it being optimised and configured for Gadi, Setonix, Nimbus with links to each infrastructure's webpage.